pybamm/util.py

```diff
@@ -57,14 +57,22 @@ def __getitem__(self, key):
         try:
             return super().__getitem__(key)
         except KeyError as error:
-            if "particle diffusivity" in key:
-                warn(
-                    f"The parameter '{key.replace('particle', 'electrode')}' "
-                    f"has been renamed to '{key}'",
-                    DeprecationWarning,
-                    stacklevel=2,
+            if "electrode diffusivity" in key or "particle diffusivity" in key:
+                old_term, new_term = (
+                    ("electrode", "particle")
+                    if "electrode diffusivity" in key
+                    else ("particle", "electrode")
```

Comment:
In which cases would we want to have the old version?

Reply:
It's to ensure the parameter name change is not breaking. This allows users to continue with the old version until we deprecate it in a future release.